src/main/resources/application.yml

```diff
@@ -0,0 +1,13 @@
+info:
```

Comment:
Controversial removing all of the properties, I have no issues, but hinders readability for debugging?